### PR TITLE
Add cloudwatch metrics to monitor fronts pressing

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?
Adding 2 metrics (pressed front json size and front decoding time) as well as refactoring front-press-latency metric so it happens in a single place.

## What is the value of this and can you measure success?
I am working on lowering the size of the pressed front files. Those metrics will allow us to monitor the impact of the changes.

## Tested in CODE?
Yes

### CODE REVIEW

File: common/app/services/S3.scala [hunk 1]
```diff
@@ -148,32 +148,17 @@ object S3FrontsApi extends S3 {
...
-    putPrivateGzipped(getDraftPressedKeyForPath(path), json, "application/json")
-
   def putLiveFapiPressedJson(path: String, json: String): Unit =
     putPrivateGzipped(getLiveFapiPressedKeyForPath(path), json, "application/json")
 
   def putDraftFapiPressedJson(path: String, json: String): Unit =
     putPrivateGzipped(getDraftFapiPressedKeyForPath(path), json, "application/json")
-
-  def getPressedLastModified(path: String): Option[String] =
-    getLastModified(getLiveFapiPressedKeyForPath(path)).map(_.toString)
```
Comment (TBonnin) [inline common/app/services/S3.scala]: Unrelated to the PR but those methods are never used anymore. Hence removing them

File: facia-press/app/frontpress/FapiFrontPress.scala [hunk 2]
```diff
@@ -117,6 +107,49 @@ trait FapiFrontPress extends Logging with ExecutionContexts {
...
+            StatusNotification.notifyFailedJob(path, isLive = isLiveContent, e)
+            throw new RuntimeException(s"${e.cause} ${e.message}")
+          },
+          _ => StatusNotification.notifyCompleteJob(path, isLive = isLiveContent))
+      )
+
+
+    pressFuture.onComplete {
+      case Success(_) =>
+        log.info(s"Successfully pressed $path in ${stopWatch.elapsed} ms")
```
Comment (jfsoul) [inline facia-press/app/frontpress/FapiFrontPress.scala]: Is it worth putting `stopWatch.elapsed` in a `val` so we guarantee all the times here are consistent.

**Review by jfsoul — APPROVED** [review]
Looks good to me.  The content size metric is long overdue! 💯

**Review by NataliaLKB — APPROVED** [review]

File: facia-press/app/frontpress/FapiFrontPress.scala (hunk 2 above)
Comment (TBonnin) [inline facia-press/app/frontpress/FapiFrontPress.scala]: yep you are right

**Pull request merged**